action_plugins/openshift_package.py

```diff
@@ -0,0 +1,98 @@
+"""
```

Comment:
nit: Unless this is an ansible-ism generally docstrings occur after the license block.